sqlite.go

```diff
@@ -993,16 +1094,45 @@ func (t *tx) exec(stmt string) (err error) {
 type conn struct {
 	*sqlite
 	*virtual.Thread
-	pdb uintptr
+	loc           *time.Location
+	compatibility string
```

Comment:
The reasoning behind this was to stay open to allow being compatible to multiple libraries.
Won't probably make sense on a second thought, so I'm changing it.